src/condor_utils/shortfile.cpp

```diff
@@ -21,8 +20,9 @@ htcondor::readShortFile( const std::string & fileName, std::string & contents )
         return false;
     }
 
-    StatWrapper sw( fd );
-    long fileSize = sw.GetBuf()->st_size;
+    struct stat statbuf = {};
+    fstat(fd, &statbuf);
+    long fileSize = statbuf.st_size;
```

Comment:
long is not 64 bits on all platforms.  use auto or filesize_t or some other type known to be 64 bits.

Reply:
Changed committed.